plugin/pkg/admission/persistentvolume/label/admission.go

```diff
@@ -40,8 +41,9 @@ var _ = admission.Interface(&persistentVolumeLabel{})
 type persistentVolumeLabel struct {
 	*admission.Handler
 
-	mutex      sync.Mutex
-	ebsVolumes aws.Volumes
+	mutex            sync.Mutex
+	ebsVolumes       aws.Volumes
+	gceCloudProvider *gce.GCECloud
```

Comment:
Fair enough.  I'm happy for us to perhaps try to take on a refactor in a separate PR, after v 1.2.